examples/enzyme/CheckboxWithLabel.js

```diff
@@ -6,15 +6,11 @@ export default class CheckboxWithLabel extends React.Component {
   constructor(props) {
     super(props);
     this.state = {isChecked: false};
```

Comment:
Shouldn't this be changed, too?

Reply:
Could do that as well, yeah (and only then #7846 can be closed).
But might as well be in a separate PR, multiple PRs for one issue keeps diffs smaller :smile:

Reply:
This is to say that this change *should* be in this PR? Let me know.

Reply:
Change added !